apps/whale-api/src/module.api/poolpair.service.ts

```diff
@@ -96,6 +98,36 @@ export class PoolPairService {
     if (b === 'DFI') {
       return info.reserveB.multipliedBy(2).multipliedBy(USDT_PER_DFI)
     }
+
+    return await this.getTotalLiquidityUsdByBestPath(info)
+  }
+
+  /**
+   * Graph based matrix resolution
+   * Returns the total liquidity of the poolpair by finding its best path to USDT
+   */
+  async getTotalLiquidityUsdByBestPath (info: PoolPairInfo): Promise<BigNumber | undefined> {
```

Comment:
Should add additional test scenarios to cover these scenarios (those that were previously included).

Reply:
Sure! Added `H-I` to test this scenario - [link](https://github.com/JellyfishSDK/jellyfish/pull/1853/files#diff-229e1b7ca9e66f6d396e594f84ff4cd6fcfb35321aca2aca2e312864279b462cR703). The ids and values adjusted too 😬 
 ![image](https://user-images.githubusercontent.com/13292662/201322037-b4000be8-118c-41e8-9871-27f34bd9e93b.png)